compiler/src/dmd/e2ir.d

```diff
@@ -6296,7 +6296,7 @@ Lagain:
     if (r == RTLSYM.MEMSET8 ||
         r == RTLSYM.MEMSET16 ||
         r == RTLSYM.MEMSET32 ||
-	r == RTLSYM.MEMSET64)
```

Comment:
Unrelated changes

Reply:
found when I ran detab over all the source files